src/open_inwoner/scss/views/App.scss

```diff
@@ -195,49 +195,49 @@
   /// Font.
 
   --font-color-heading: var(--color-black);
-  --font-family-heading: 'Heading';
+  // Font-family = 'Heading' coming from both NLDS and/or the font-upload functionality
+  --font-family-heading: var(--utrecht-heading-font-family);
   --font-weight-heading: bold;
 
   --font-color-heading-1: var(--font-color-heading);
   --font-family-heading-1: var(--font-family-heading);
   --font-line-height-heading-1: 41px;
-  --font-size-heading-1: 32px;
+  --font-size-heading-1: var(--utrecht-heading-1-font-size);
   --font-weight-heading-1: var(--font-weight-heading);
 
   --font-color-heading-2: var(--font-color-heading);
   --font-family-heading-2: var(--font-family-heading);
   --font-line-height-heading-2: 34px;
-  --font-size-heading-2: 24px;
+  --font-size-heading-2: var(--utrecht-heading-2-font-size);
   --font-weight-heading-2: var(--font-weight-heading);
 
   --font-color-heading-3: var(--font-color-heading);
   --font-family-heading-3: var(--font-family-heading);
   --font-line-height-heading-3: 30px;
-  --font-size-heading-3: 20px;
+  --font-size-heading-3: var(--utrecht-heading-3-font-size);
   --font-weight-heading-3: var(--font-weight-heading);
 
   --font-color-heading-4: var(--font-color-heading);
   --font-family-heading-4: var(--font-family-heading);
   --font-line-height-heading-4: 26px;
-  --font-size-heading-4: 16px;
+  --font-size-heading-4: var(--utrecht-heading-4-font-size);
   --font-weight-heading-4: var(--font-weight-heading);
 
   --font-size-heading-card: 18px;
 
-  --font-family-body: 'Body';
+  // Font-family = 'Body' coming from both NLDS and/or the font-upload functionality
+  --font-family-body: var(--utrecht-document-font-family);
   --font-color-body: var(--color-gray-dark);
-  --font-size-body: 16px;
+  --font-size-body: var(--utrecht-document-font-size);
   --font-line-height-body: 24px;
 
   --font-size-body-extra-small: 12px;
-  --font-size-body-small: 14px;
+  --font-size-body-small: var(--utrecht-paragraph-small-font-size);
   --font-size-body-large: 20px;
```

Comment:
Why is this hard-coded (it's the only case of font size not defined in terms of utrecht classes)?

Reply:
Do you mean the '--font-size-body-large'? I did not change anything in this PR there. 
Also: no this value does not exist in NLDS for the Document level.
The NLDS Document only has a couple of attributes and values; all other modifiers I guess should belong to actual components. 

If you look at Utrecht's Storybook here https://nl-design-system.github.io/utrecht/storybook/?path=/docs/css_css-document--docs you can inspect their Document/paragraph component, which will point to a long file with all of their CSS variables. It does not contain many `-document-...` variables.

Utrecht does have a variable '--utrecht-typography-scale-lg-font-size' but that is not part of the NLDS standard, and is a 'scaling' variable font-size so not the same as our set pixel size.

Additionally: this '--font-size-body-large' variable is only used in OIP for: just a few icons, 2 asterisks and the mobile menu... But currently the accessibility header is not visible on mobile so that is outside of the scope here. 
(That is a future task that needs a new design).
➕ Forgive me for not adding an enlarger to a font-size that is already 'large' 😉 which is why I also omitted H1 and H2 since those are already quite large.